examples/ui-demo/src/components/small-cards/SolanaCard.tsx

```diff
@@ -0,0 +1,167 @@
+import { queryClient } from "@/app/config";
+import { useToast } from "@/hooks/useToast";
+import { useSigner, useSignerStatus } from "@account-kit/react";
+import { LAMPORTS_PER_SOL } from "@solana/web3.js";
+import { useMutation, useQuery } from "@tanstack/react-query";
+import { useMemo } from "react";
+import { toBytes, toHex } from "viem";
+import { LoadingIcon } from "../icons/loading";
+import { UserAddressTooltip } from "../user-connection-avatar/UserAddressLink";
+import { Button } from "./Button";
+import { createTransfer } from "./solana/createSolanaTransfer";
+import * as solanaNetwork from "./solana/solanaNetwork";
+
+export const SolanaCard = () => {
+  const signer = useSigner();
+  const status = useSignerStatus();
+  const solanaSigner = useMemo(() => {
+    if (!signer) return;
+    if (!status.isConnected) return;
+    return signer.experimental_toSolanaSigner();
+  }, [signer, status.isConnected]);
+  const connection = solanaNetwork.connect();
```

Comment:
[nit: fine to keep calling this w/ every re-render or should we wrap it in a `useState` w/ an init function?nit: fine to keep calling this w/ every re-render or should we wrap it in a `useState` w/ an init function?](nit: fine to keep calling this w/ every re-render or should we wrap it in a `useState` w/ an init function?)

Reply:
ah yea lemme move that, honestly this could just happen outside of the component